GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs

```diff
@@ -133,52 +121,121 @@ private static void UpdateGuiFocus(ToolBarSelectedButton currentTab)
 
         private void OnGuiCreateBundleSelect()
         {
-            EditorGUILayout.LabelField("AssetBundle Creation", EditorStyles.boldLabel);
+            var descriptionTextStyle = new GUIStyle(GUI.skin.label)
```

Comment:
Create a method: CreateDescriptionTextStyle() and use it in place of this and the 3 other cases.

Reply:
Resolved.